From c24326067a44c306eaf69b0e6e93c403c0d39ed7 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:45 -0400
Subject: [redhat] kabi: Adding symbol init_task (init/init_task.c)

Message-id: <9a10ebcb03537568f501d80b9e4dc67482b6c5f2.1410510773.git.poros@redhat.com>
Patchwork-id: 96112
O-Subject: [PATCH RHEL7.1 105/233] kabi: Adding symbol init_task (init/init_task.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Used to set default task attributes from the initial task struct, primarily the defaults in struct cred.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/init_task
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/init_task
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/init_task

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/init_task b/redhat/kabi/kabi-module/kabi_ppc64/init_task
new file mode 100644
index 0000000..afc8a7c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/init_task
@@ -0,0 +1,2 @@
+#1-
+0x7951b3f0 init_task vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/init_task b/redhat/kabi/kabi-module/kabi_s390x/init_task
new file mode 100644
index 0000000..f09e76b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/init_task
@@ -0,0 +1,2 @@
+#1-
+0x69e94571 init_task vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/init_task b/redhat/kabi/kabi-module/kabi_x86_64/init_task
new file mode 100644
index 0000000..c4dd668
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/init_task
@@ -0,0 +1,2 @@
+#1-
+0x6be0bc63 init_task vmlinux EXPORT_SYMBOL
-- 
1.7.1